src/hipscat_import/dask_map_reduce.py

```diff
@@ -0,0 +1,99 @@
+"""Import a set of non-hipscat files using dask for parallelization"""
+
+import hipscat.io.write_metadata as io
+import hipscat.pixel_math as pixel_math
+import numpy as np
+from dask.distributed import Client, progress, wait
+
+import hipscat_import.map_reduce as mr
+from hipscat_import.arguments import ImportArguments
+
+
+def _generate_histogram(args, client):
```

Comment:
this could be a future task to replace and I see that this one is parallelized, but is this histogram method redundant with the one found in the `hipscat` package?

Reply:
This one should really be _map, since it's doing more than just generating a histogram of object counts (also makes it parallel with the below _reduce)

The one in hipscat is *only* getting the counts of objects per-pixel.